packages/components/src/Slider/Slider.component.js

```diff
@@ -254,7 +264,16 @@ class Slider extends React.Component {
 						min={min}
 						max={max}
 						handle={noValue ? undefined : this.state.handle}
-						className={classnames(theme['tc-slider-rc-slider'], 'tc-slider-rc-slider')}
+						className={classnames(
```

Comment:
Could you use getTheme like in 
https://github.com/Talend/ui/blob/cbd4f128b72705c7a7f3ea98cdf8bb1a96941ca6/packages/components/src/PieChart/PieChartIcon.component.js#L95

It allows to duplicate the classnames declaration :)

Reply:
I'll take a look!